python/mxnet/ndarray/contrib.py

```diff
@@ -18,9 +18,81 @@
 # coding: utf-8
 # pylint: disable=wildcard-import, unused-wildcard-import
 """Contrib NDArray API of MXNet."""
+import math
+from ..context import current_context
+from ..random import uniform
 try:
     from .gen_contrib import *
 except ImportError:
     pass
 
-__all__ = []
+__all__ = ["log_uniform_candidate_sampler"]
+
+# pylint: disable=line-too-long
+def log_uniform_candidate_sampler(true_classes, num_sampled, range_max, ctx=None):
```

Comment:
should it go under contrib.random? since other sampling methods in random just have the distribution as name, should we follow the same convention?

Reply:
Name changed to rand_zipfian to follow the convention. Extra namespaces such as `contrib.random` might over-complicate APIs since there are just a few operators in nd.contrib.